bindings/python/htcondor2/_schedd.py

```diff
@@ -294,6 +320,142 @@ def act(self,
 
         return pyResult
 
+    def addUserRec(self,
+        user_spec : Union[List[str], str, List[classad.ClassAd] ],
+    ) -> classad.ClassAd:
+        """
+        Add User record(s) to the *condor_schedd* daemon if a User
```

Comment:
Nitpick: the capital `U` here (and immediately after) look a little weird to me.  It's not consistent with `user_spec` being all lower-case (probably OK, that's a Python convention) or with "Which user(s) to add."